includes/misc-functions.php

```diff
@@ -1610,11 +1610,18 @@ function give_delete_meta( $id, $meta_key, $meta_value = '' ) {
  * @return bool                   If the action has been added to the completed actions array
  */
 function give_has_upgrade_completed( $upgrade_action = '' ) {
-
+	// Bailout.
 	if ( empty( $upgrade_action ) ) {
 		return false;
 	}
 
+	// Fresh install?
+	// If fresh install then all upgrades will be consider as completed.
+	if ( ! get_option( 'give_version' ) ) {
```

Comment:
@DevinWalker On fresh install if anybody uses it before plugin activates hook then it will return false but for the fresh install, all upgrades will complete by default, so I updated this function.

Reply:
Using `get_option` within a conditional will cause a PHP fatal write error in old versions of PHP. You have to save it to a variable before.

Reply:
Fixed https://github.com/WordImpress/Give/pull/2233/commits/4f52aaa6510999e6c6dfaed790803e28adb3e793